pyseir/inference/model_fitter.py

```diff
@@ -35,6 +32,61 @@ def calc_chi_sq(obs, predicted, stddev):
     return np.sum((obs - predicted) ** 2 / stddev ** 2)
 
 
+@lru_cache(maxsize=None)
+def load_pyseir_fitter_initial_conditions_df():
+    return pd.read_csv(
+        "./pyseir_data/pyseir_fitter_initial_conditions.csv", dtype={"fips": str}
+    ).set_index("fips")
+
+
+@dataclass(frozen=True)
+class RegionalInput:
```

Comment:
`RegionalModelFitterInput`? `RegionalFitterInput`? not sure if it's as necessary since its in this module

Reply:
Obviously it'll be referred to as model_fitter.RegionalInput if ever used in another module :-) so I think the current name is specific enough.